bluesky/protocols.py

```diff
@@ -192,18 +265,18 @@ def unstage(self) -> List[Any]:
 
 @runtime_checkable
 class Pausable(Protocol):
-    def pause(self) -> Union[NoReturn, None]:
+    def pause(self) -> SyncOrAsync[None]:
```

Comment:
Easier for the RE to make these `SyncOrAsync` rather than `Status`. Is there a downside to it?